### DISCUSSION TITLE
When update????????

### DISCUSSION THREAD

**Discussion by courpiaque** [dotnet/maui#3570] [Q&A]
Shell is broken as hell

**Comment by nogginbox** [comment]
_I'm struggling with theming shell at the moment. I can't get TitleView to work, but maybe it's broken and not me doing it wrong._

**Anyway,** there is an update once a month. The last one for Preview 10 was on 8 November 2021, so the next one should be early in December.

You can see the **[Maui Roadmap](/dotnet/maui/wiki/Roadmap)** to see what they're focusing on.

**Reply by nogginbox** [reply]
You can also see the status of various elements of MAUI on the [MAUI Development Status Page](https://github.com/dotnet/maui/wiki/Status), although I'm not sure if this is off the current release or of the latest changes in the repo.

**Comment by VladislavAntonyuk** [comment]
December 14

**Reply by nogginbox** [reply]
Has there been a delay?

**Reply by VladislavAntonyuk** [reply]
the packages are already available: https://dev.azure.com/dotnet/CommunityToolkit/_build/results?buildId=62664&view=results

**Reply by nogginbox** [reply]
Are there any instructions on how to use the packages?

I was only able to update to MAUI Preview 10 by updating Visual Studio Preview.

**Reply by VladislavAntonyuk** [reply]
run `dotnet workload update` or `dotnet workload install maui`:
![image](https://user-images.githubusercontent.com/33021114/146149857-f51bfa54-7a3d-4690-bdcf-c39d491508e4.png)

**Reply by nogginbox** [reply]
Thanks @VladislavAntonyuk I'm now on preview 11 and it's fixed a few issues I was having. They're very quiet about this update.
